app/bundles/UserBundle/Resources/views/Profile/index.html.twig

```diff
@@ -64,25 +64,28 @@
                     </div>
                     <div class="row">
                         <div class="col-md-6">
-                            <h3 id="personal-info" class="pt-20">{{ 'mautic.user.config.title.get_to_know_you'|trans }}</h3>
+                            <h3 id="personal-info" class="pt-20 mb-xs">{{ 'mautic.user.config.title.get_to_know_you'|trans }}</h3>
                             <div class="text-muted small description pb-20">{{ 'mautic.user.config.personalize.interface.helper'|trans }}</div>
                             {{ permissions.editName ? form_row(userForm.firstName) : form_row(userForm.firstName_unbound) }}
                             {{ permissions.editName ? form_row(userForm.lastName) : form_row(userForm.lastName_unbound) }}
                             {{ permissions.editPosition ? form_row(userForm.position) : form_row(userForm.position_unbound) }}
                             {{ form_row(userForm.signature) }}
 
-                            <h3 id="locale" class="pt-20">{{ 'mautic.user.config.title.locale'|trans }}</h3>
+                            <h3 id="locale" class="pt-20 mb-xs">{{ 'mautic.user.config.title.locale'|trans }}</h3>
                             <div class="text-muted small description pb-20">{{ 'mautic.user.config.region.preferences.helper'|trans }}</div>
                             {{ form_row(userForm.timezone) }}
                             {{ form_row(userForm.locale) }}
                         </div>
                         <div class="col-md-6">
-                            <h3 id="security" class="pt-20">{{ 'mautic.user.config.title.account_data_security'|trans }}</h3>
+                            <h3 id="security" class="pt-20 mb-xs">{{ 'mautic.user.config.title.account_data_security'|trans }}</h3>
                             <div class="text-muted small description pb-20">{{ 'mautic.user.config.account.security.helper'|trans }}</div>
                             {{ permissions.editUsername ? form_row(userForm.username) : form_row(userForm.username_unbound) }}
                             {{ permissions.editEmail ? form_row(userForm.email) : form_row(userForm.email_unbound) }}
                             {{ form_row(userForm.plainPassword.password) }}
                             {{ form_row(userForm.plainPassword.confirm) }}
+                            <h3 id="security" class="pt-20 mb-xs">{{ 'mautic.user.config.title.experience_and_learning'|trans }}</h3>
```

Comment:
The new h3 should have it's own id.

Reply:
Really nice catch, thanks for reviewing!